<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>817</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dynamic Pattern</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>781</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Shape</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Radius</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Color</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Position</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="open_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_length">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1920</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_width">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1080</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cross_line_click">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cross Line</string>
    </property>
   </widget>
   <widget class="MyView" name="DynamicPattern">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>642</width>
      <height>362</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::NoDrag</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_import"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_import">
   <property name="text">
    <string>Import </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mainUI.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>get_data_from_tabelwidget()</slot>
  <slot>cell_entered_test()</slot>
 </slots>
</ui>
